packages/prosemirror-lwdita/src/request.ts

```diff
@@ -98,13 +108,20 @@ export function processRequest(): undefined | { key: string, value: string }[] {
     const currentUrl = window.location.href;
 
     try {
-      const parameters = getParameterValues(currentUrl);
-      showNotification(parameters);
-      console.log('Parameters =', JSON.stringify(parameters));
-      if(typeof parameters === 'object') {
-        return parameters;
+      const parameters = getAndValidateParameterValues(currentUrl);
+      if (typeof parameters === 'string') {
+        showNotification(parameters);
       }
-      
+
+      if (typeof parameters === 'object') {
+        // Store the parameters in localStorage for reading the values after the OAuth flow
+        // TODO: After a successful GitHub Authentication, read the user parameters from localStorage and clear it afterwards
+        localStorage.setItem('userParams', JSON.stringify(parameters));
```

Comment:
These params should be passed to GitHub and back using `redirect_uri`, see https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/authorizing-oauth-apps#1-request-a-users-github-identity for more info.
No need to store them in the `localStorage`.

Reply:
This bit will be moved to the encoding task that will cover `redirect_uri` and `state`.